src/MagnumPlugins/TinyGltfImporter/Test/mesh-custom-attributes.gltf

```diff
@@ -4,16 +4,27 @@
     },
     "meshes": [
         {
+            "name": "standard types",
             "primitives": [
                 {
                     "attributes": {
                         "_TBN": 0,
                         "_UV_ROTATION": 1,
                         "_TBN_PRECISER": 2,
-                        "_OBJECT_ID3": 3,
-                        "_DOUBLE_SHOT": 4,
-                        "_NEGATIVE_PADDING": 5,
-                        "_NOT_AN_IDENTITY": 6
+                        "_OBJECT_ID3": 3
+                    }
+                }
+            ]
+        },
+        {
+            "name": "non-standard types",
+            "note": "attribute types not allowed by the spec but accepted by tinygltf",
+            "primitives": [
+                {
+                    "attributes": {
+                        "_NEGATIVE_PADDING": 4,
+                        "NOT_AN_IDENTITY": 5,
```

Comment:
Does the lack of a leading underscore have any meaning or is it just random? Like, cgltf only ever parsing custom attributes that *do* begin with an underscore? Or, in other words, if the "standard type" section above would have one of them with a leading underscore, would cgltf die on that?

Reply:
Technically custom attributes without underscore are not allowed, but neither tinygltf nor cgltf care. In `CgltfImporter` I added a warning *"unknown attribute NOT_AN_IDENTITY, importing as custom attribute"*. But yeah, it could as well go into the "standard type" mesh if it wasn't for the double component type (5130). The warning I mentioned above happens in `doOpenData` so this works for testing in `CgltfImporter`, but I'll change it to have one attribute without underscore in "standard types" too. Good catch 😊